packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -143,6 +143,10 @@ export class Meter implements types.Meter {
     metric: Metric<T>
   ): void {
     if (this._metrics.has(name)) {
+      // @todo: decide how to handle already registered metric
```

Comment:
Should we make an issue to track/discuss this question?

Reply:
done.